plutus-core/testlib/PlutusCore/Generators/QuickCheck/Builtin.hs

```diff
@@ -244,7 +244,9 @@ instance Arbitrary Data where
     arbitrary = arbitraryBuiltin
     shrink = shrinkBuiltin
 
-deriving via Map ByteString (Map ByteString Integer) instance Arbitrary Value
+instance Arbitrary Value where
+    arbitrary = Value.pack <$> arbitrary
```

Comment:
This is a bad generator. In particular, you're very unlikely to generate two different `Value`s that share the same non-empty token name or currency symbol. See [this](https://github.com/IntersectMBO/plutus/blob/8d810d47112a6ca6a995feb968ffab99c622f217/plutus-ledger-api/testlib/PlutusLedgerApi/Test/V1/Data/Value.hs) for how we do it in `plutus-ledger-api`.

Reply:
Agreed - I'm going to create a different (hedgehog) generator in the next PR.